CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.26.4] - 2022-01-24
+- Modified the service-cluster mapping for regular service Znodes to be consistent with the change in d2-config-service.
```

Comment:
Let's not mention d2-config-service as this is a LI-internal MP.
I would prefer `Map local variant of service ZNodes to cluster without colo suffix`